code/modules/mob/new_player/sprite_accessories/sprite_accessories.dm

```diff
@@ -581,7 +583,7 @@
 /datum/sprite_accessory/socks/thin_knee
 	name = "Knee-high Thin"
 	icon_state = "thin_knee"
-	gender = FEMALE
+	body_type = FEMALE
```

Comment:
I wonder if these are still necessary, servers like TG have them applicable to all genders, and I don't think they have gender specific sprites

Reply:
It's necessary for at least some undergarments, where the differences in body shape mean the sprites don't fit *quite* right. Socks and stuff are probably fine for both.